lib_eio/core/eio__core.mli

```diff
@@ -265,27 +265,29 @@ module Fiber : sig
       The current task remains runnable, but goes to the back of the queue.
       Automatically calls {!check} just before resuming. *)
 
-  (** {2 Concurrent list operations} *)
-
-  val filter : ?max_fibers:int -> ('a -> bool) -> 'a list -> 'a list
```

Comment:
Would be nice to mark these as deprecated first, with a clue where they went, for one release at least.

Reply:
done